src/i18n/strings/en_EN.json

```diff
@@ -778,8 +778,10 @@
     "When rooms are upgraded": "When rooms are upgraded",
     "My Ban List": "My Ban List",
     "This is your list of users/servers you have blocked - don't leave the room!": "This is your list of users/servers you have blocked - don't leave the room!",
-    "Active call": "Active call",
-    "Call Paused": "Call Paused",
+    "Video Call": "Video Call",
+    "Voice Call": "Voice Call",
+    "Fill screen": "Fill screen",
```

Comment:
It seems inconsistent that "Fill screen" only capitalises the first word, while e.g. "Video Call" capitalises both.

Reply:
hmm, yeah - these are hovers so aren't in the designs, ie. I'm just making them up, so let's go with that.